@kiva/kv-components/vue/KvTextInput.vue

```diff
@@ -45,6 +45,16 @@
 				:icon="mdiAlertCircleOutline"
 				class="tw-absolute tw-top-1.5 tw-right-1.5 tw-pointer-events-none tw-text-danger"
 			/>
+			<button
+				v-if="canClear && valid"
```

Comment:
We should only show this button if there is text in the input. Sorry that wasn't specced out.

Reply:
Done